### PR TITLE
Update graph automatically

### LINKED ISSUE #33: Update graph automatically
Repeatedly update the graph in the backend. We could use https://fastapi-utils.davidmontague.xyz/user-guide/repeated-tasks/ to trigger the `update` endpoint every X minutes.

### PR DESCRIPTION
**Pull request by VictorKuenstler**
Closes #33

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -88,3 +88,7 @@ wcwidth==0.2.5
 websocket-client==0.57.0
 websockets==8.1
 zipp==3.4.0
+fastapi-utils==0.2.1
```
Comment (philipp94831) [inline backend/requirements.txt:91]: isn't there a lock file or sth that is update everytime? Also the new lines are unnecessary but not too important ;-)

**Review by philipp94831 — COMMENTED** [review]
What about a test?

File: backend/requirements.txt (hunk 1 above)
Comment (disrupted) [reply backend/requirements.txt:91]: luckily not for Python :)

File: backend/settings.yaml [hunk 2]
```diff
@@ -4,6 +4,9 @@ kafkaconnect:
     - name: "Transformer"
       key: "transforms.changeTopic.regex"
 
+# update graph (fetch deployments and connectors) every X seconds
```
Comment (disrupted) [inline backend/settings.yaml:7]: Can you add this to the Readme?

**Review by disrupted — CHANGES_REQUESTED** [review]

**Review by disrupted — APPROVED** [review]
LGTM

**Comment by VictorKuenstler** [comment]
> What about a test?

I added a test that should cover this.

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**